tensorflow_io/cifar/python/ops/cifar_dataset_ops.py

```diff
@@ -0,0 +1,148 @@
+# Copyright 2018 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""CIFAR Dataset."""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import tensorflow
+from tensorflow import dtypes
+from tensorflow.compat.v1 import data
+from tensorflow_io import _load_library
+cifar_ops = _load_library('_cifar_ops.so')
+
+class _CIFAR10Dataset(data.Dataset):
```

Comment:
Could you clarify the differences between the class whose name starts with underscore and the other class? Maybe it would be better to combine them into one class?